app/controllers/foreman_plugin_template/react_controller.rb

```diff
@@ -0,0 +1,17 @@
+module ForemanPluginTemplate
+  class ReactController < ::ApplicationController
```

Comment:
Should plugins reuse the `ReactController` and react layout from core?

Reply:
When the react router for plugins will be ready in foreman-core, the core's react layout and controller will be reusable, until then we still need to mount plugin's root component via plugin's react layout.
```ruby
# plugin/layout/react
mount_react_component('PluginTemplate', '#foremanPluginTemplateRoot')
```

Reply:
So can we drop this now?